### PR TITLE
Added Left and Right Wheel Velocities

### PR DESCRIPTION
**Pull request by npochhi**

### CODE REVIEW

**Comment by SpandanKumarSahu** [comment]
LGTM !

**Comment by pranitbauva1997** [comment]
@SpandanKumarSahu Please give enough time for others to review the code before merging.

File: Motion.h [hunk 1]
```diff
@@ -25,7 +26,7 @@ void forward(int BotID,int t=10){
   }
   if(t>10)
     t=10;
-
+
```
Comment (pranitbauva1997) [inline Motion.h:29]: Extra.

File: test.cpp [hunk 2]
```diff
@@ -13,11 +13,8 @@ int main(int argcs, char** argv){
   vector<SwarmBot>bots;
   BOT_ID=atoi(argv[1]);
   while(true){
-    bots=getAllPose();
-    if(BOT_ID%2 ==0)
-      forward(BOT_ID);
-    else
-      backward(BOT_ID);
+		 bots=getAllPose();
```
Comment (pranitbauva1997) [inline test.cpp:16]: Please keep the indentation proper.

File: Motion.h [hunk 3]
```diff
@@ -143,5 +144,66 @@ void leftTurn(int BotID, int t=10){
...
+    t=10;
+
+	float velLeft = leftRPM * WHEEL_RADIUS_CONSTANT;
+	float velRight = rightRPM * WHEEL_RADIUS_CONSTANT;
+	//Assuming rotation around a fixed point (JEE ki yaad dila di! :D )
+	float angVel = (velRight - velLeft) / (2 * BotSize);
+	float r = static_cast <float> (rand()) / static_cast <float> (RAND_MAX);
+
+	if(velLeft != velRight) {
+		float AngRadius = (velLeft + velRight) * BotSize / (velRight - velLeft);
```
Comment (pranitbauva1997) [inline Motion.h:167]: Please keep the indentation proper.

**Review by pranitbauva1997 — COMMENTED** [review]
Please don't add binary files into the commit. Adding that is *wrong*.

**Comment by pranitbauva1997** [comment]
@SpandanKumarSahu Please give enough time for others to review the code before merging.

**Comment by pranitbauva1997** [comment]
I haven't gone through the source code thoroughly because I am assuming @SpandanKumarSahu has. If not, then I guess we can revert all of this and then work on this again.

**Comment by SpandanKumarSahu** [comment]
@pranitbauva1997 My bad!
Will take care of this next time.
The code was correct, and I mean the working part of it.

**Pull request merged**